src/i18n/resources/en.js

```diff
@@ -1054,43 +1054,44 @@ The FRA team
   },
 
   editUser: {
+    activate: 'Activate',
+    activated: 'Activated',
     chooseProfilePicture: 'Choose picture',
-    title: 'Appellation',
+    city: 'City',
+    contactPreference: 'Preferred way of contacting',
+    contactPreferenceMethod: 'Contacting method',
+    contribution: 'Contribution',
+    countryIso: 'Country',
+    deactivate: 'Deactivate',
+    demoteToUser: 'Are you sure you want to remove Admin privileges?',
+    disabled: 'Disabled',
+    done: 'Save',
+    email: 'Email',
+    institution: 'Institution',
+    loginEmail: 'Login',
+    mandatoryFields: '* are mandatory fields',
     name: 'Name',
-    surname: 'Surname(s)',
-    professionalTitle: 'Professional title',
-    organizationalUnit: 'Organizational unit',
     organization: 'Organization',
-    street: 'Street address',
-    zipCode: 'Zip code',
+    organizationalUnit: 'Organizational unit',
+    picture1MbMax: 'Profile picture cannot exceed 1MB',
+    platformChat: 'PlatformChat',
     poBox: 'P.O. Box',
-    city: 'City',
-    countryIso: 'Country',
+    position: 'Position',
     primaryEmail: 'Primary email address',
-    secondaryEmail: 'Secondary email address',
     primaryPhoneNumber: 'Primary telephone number',
+    professionalTitle: 'Professional title',
+    promoteToAdmin: 'Are you sure you want to grant Admin privileges?',
+    role: 'Role',
+    secondaryEmail: 'Secondary email address',
     secondaryPhoneNumber: 'Secondary telephone number',
-    skype: 'Skypename',
-    contactPreference: 'Preferred way of contacting',
-    contactPreferenceMethod: 'Contacting method',
-    platformChat: 'PlatformChat',
     signal: 'Signal',
-    whatsapp: 'Whatsapp',
-    role: 'Role',
-    email: 'Email',
-    loginEmail: 'Login',
-    institution: 'Institution',
-    position: 'Position',
-    done: 'Save',
-    deactivate: 'Deactivate',
-    activate: 'Activate',
-    picture1MbMax: 'Profile picture cannot exceed 1MB',
-    activated: 'Activated',
+    skype: 'Skypename',
     status: 'Status',
-    demoteToUser: 'Are you sure you want to remove Admin privileges?',
-    promoteToAdmin: 'Are you sure you want to grant Admin privileges?',
-    mandatoryFields: '* are mandatory fields',
-    disabled: 'Disabled',
+    street: 'Street address',
+    surname: 'Surname(s)',
+    title: 'Appellation',
+    whatsapp: 'Whatsapp',
+    zipCode: 'Zip code',
   },
 
```

Comment:
I have added missing translation (`Contribution`, other are only sorted) to the main PR todo
Hopefully it's ok I will ask it as last step, in case we need to add other (eg. Mr, Mrs...)